### PR TITLE
remove <meta httpequiv="Content-Security-Policy"> from markup (close #243)

### LINKED ISSUE #243: Add support sites with 'Content-Security-Policy'
'Content-Security-Policy' header allows to restrict resouce loading (scripts, styles).
After proxing a same sites we can have a 'Refused to load the script ' errors.

Example - https://www.dropbox.com/

Details:
http://www.w3.org/TR/CSP1/
https://en.wikipedia.org/wiki/Content_Security_Policy
http://content-security-policy.com/
http://www.html5rocks.com/en/tutorials/security/content-security-policy/

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @inikulin @churkin 
@VasilyStrelyaev check test name and comments

### CODE REVIEW

File: src/processing/dom/index.js [hunk 1]
```diff
@@ -216,6 +216,11 @@ export default class DomProcessor {
         this.adapter.setAttr(el, 'autocomplete', 'off');
     }
 
+    _processContentSecurityPolicy (el) {
```
Comment (inikulin) [inline src/processing/dom/index.js]: I believe it should be done in context of `_processMetaElement`

File: src/processing/dom/index.js [hunk 2]
```diff
@@ -216,6 +216,11 @@ export default class DomProcessor {
...
+        if (this.adapter.getAttr(el, 'http-equiv').toLowerCase() === 'content-security-policy')
+            this.adapter.removeElement(el);
```
Comment (inikulin) [inline src/processing/dom/index.js]: Is it ok to remove element during traversing? Can it potentially brake anything? \cc @churkin

**Comment by inikulin** [comment]
\r-

File: src/processing/dom/index.js (hunk 2 above)
Comment (churkin) [inline src/processing/dom/index.js]: @inikulin right, you remove the array item in the foreach loop

Comment (inikulin) [inline src/processing/dom/index.js]: Maybe we should remove meta attributes instead, so it will be just skiped by the browser?

**Comment by VasilyStrelyaev** [comment]
lgtm

File: src/processing/dom/index.js (hunk 2 above)
Comment (miherlosev) [inline src/processing/dom/index.js]: > @inikulin right, you remove the array item in the foreach loop

we use `for` loop for selector patterns - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/processing/dom/index.js#L171
We don't remove the array item in the foreach loop

Comment (miherlosev) [inline src/processing/dom/index.js]: ping @inikulin @churkin

**Comment by inikulin** [comment]
@miherlosev What about https://github.com/DevExpress/testcafe-hammerhead/pull/246#discussion_r44264169 ?

**Comment by miherlosev** [comment]
https://github.com/DevExpress/testcafe-hammerhead/pull/246#discussion_r44267905

File: src/processing/dom/index.js (hunk 2 above)
Comment (inikulin) [inline src/processing/dom/index.js]: We need to use reverse for-loop then.

**Comment by miherlosev** [comment]
ping @inikulin @churkin

**Comment by inikulin** [comment]
@miherlosev https://github.com/DevExpress/testcafe-hammerhead/pull/246#discussion_r44280991

**Comment by miherlosev** [comment]
After discussing with @inikulin  we decide to remove `http-equiv` and `content` attributes from the `meta` tag instead of removing a whole element.

**Comment by miherlosev** [comment]
FPR

**Comment by miherlosev** [comment]
ping @inikulin @churkin

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**